### ISSUE TITLE
Autocloaking won't turn off

### ISSUE THREAD

**Issue by MrCodingFanboy** [dotenv-org/dotenv-vscode#85]
Basically the title.
After the install the values were cloaked, but when I want to toggle it - with the button in the file, the command palette or disabling the feature in the settings - it won't turn off and I can't change the values around.

**Comment by navin-moorthy** [comment]
I am also facing the same issue !

**Comment by PaulMEdwards** [comment]
Same. I had to disable it in both `User` & `Workspace` settings.

**Comment by francmarin98** [comment]
Yes, toggle not works in the last version. Thanks @gudbrandtandberg works fine :D

**Comment by dragon-fish** [comment]
FYI https://github.com/dotenv-org/dotenv-vscode/issues/29

Adding these lines to `settings.json` works for me:

```js
{
  // ...
  "editor.tokenColorCustomizations": {
     "textMateRules": []
  }
}
```

**Comment by AngeloCore** [comment]
Bump issue

**Comment by motdotla** [comment]
Thank you for raising this. We'll fix up soon.

**Comment by roguebaires** [comment]
For anyone using v0.23.0 downgrade to v.0.22.0 to fix the toggle issue and update until v != 0.23.0

**Comment by motdotla** [comment]
We just published `v0.24.1`, which should fix this problem. Let me know how it goes for you.

**Comment by Newtoniano** [comment]
@motdotla doesn't seem to be working, at least with version `v0.24.2` in vscode-insiders. I get `Cannot set properties of undefined (setting 'textMateRules')`

**Comment by motdotla** [comment]
This should be patched in the just released `0.24.3`. Please install that and reload and try again. Sorry about this!
